packages/block-editor/src/components/skip-to-selected-block/index.js

```diff
@@ -28,8 +28,7 @@ export default function SkipToSelectedBlock() {
 
 	return selectedBlockClientId ? (
 		<Button
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
+			__next40pxDefaultSize
```

Comment:
Insert a Paragraph block, and hit <kbd>Tab</kbd> all the way through the block inspector sidebar until this skip button pops up.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/12b12770-b5e0-482c-ae66-c1c1deedee63" alt="Skip to selected block button, before" width="276">|<img src="https://github.com/user-attachments/assets/d85d2825-2dd5-4613-8527-87bbd848527a" alt="Skip to selected block button, after" width="276">|